dagger/storageproofs/rsa.nim

```diff
@@ -0,0 +1,195 @@
+## Nim-POS
+## Copyright (c) 2021 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+import libp2p/crypto/crypto # for RSA
+import bearssl
+import nimcrypto # for SHA512
+import random
```

Comment:
unless for generating random tests, we should never import nim std/random in a crypto-sensitive library.

Reply:
Right, I will remove these using only the bearssl Rng